test/utils/image/manifest.go

```diff
@@ -214,7 +217,7 @@ func initImageConfigs() map[int]Config {
 	configs[CudaVectorAdd2] = Config{e2eRegistry, "cuda-vector-add", "2.0"}
 	configs[DebianIptables] = Config{buildImageRegistry, "debian-iptables", "v12.1.0"}
 	configs[EchoServer] = Config{e2eRegistry, "echoserver", "2.2"}
-	configs[Etcd] = Config{gcRegistry, "etcd", "3.4.7"}
+	configs[Etcd] = Config{prodRegistry, "etcd", "3.4.9"}
```

Comment:
IN order to unblock this change, can we promote it to the old repo in the meantime and keep using that for a bit *until the flip will really happen"?

Reply:
@dims @justaugustus If we want to use the old prod repo, is there a way to promote image from new staging repo to old prod repo? Or do we have to manually push to old staging repo first?

Reply:
From what I know the latter only (you need to push to the old staging repo)